src/main/java/lt/vu/web/api/v1/dto/user/UpdateUserDTO.java

```diff
@@ -0,0 +1,41 @@
+package lt.vu.web.api.v1.dto.user;
+
+import lombok.Getter;
+import lombok.Setter;
+import lt.vu.infrastructure.validators.PhoneNumber;
+import lt.vu.application.user.service.UserInfoDTO;
+import lt.vu.web.api.v1.dto.address.PostAddressDTO;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import javax.ws.rs.FormParam;
+
+@Getter @Setter
+public class UpdateUserDTO implements UserInfoDTO {
```

Comment:
I think we should stick to the usual way we name DTOs - `<HTTP method name><model name>DTO`. In this case - `PutUserInfoDTO`.